queries.sql

```diff
@@ -1,3 +1,34 @@
-/*Queries that provide answers to the questions from all projects.*/
+/*Queries that provide answers to the questions from all projects.
+Find all animals whose name ends in "mon".
+List the name of all animals born between 2016 and 2019.
+List the name of all animals that are neutered and have less than 3 escape attempts.
+List the date of birth of all animals named either "Agumon" or "Pikachu".
+List name and escape attempts of animals that weigh more than 10.5kg
+Find all animals that are neutered.
+Find all animals not named Gabumon.
+Find all animals with a weight between 10.4kg and 17.3kg (including the animals with the weights that equals precisely 10.4kg or 17.3kg)
+*/
 
-SELECT * from animals WHERE name = 'Luna';
+-- Find all animals whose name ends in "mon".
+SELECT name from animals WHERE name like '%mon';
+
+-- List the name of all animals born between 2016 and 2019.
+SELECT name from animals WHERE date_of_birth >= '2016-01-10' and date_of_birth < '2019-12-31';
+ 
```

Comment:
- [ ] Please change this query to select all animals born between 2016 and 2019. Currently, this query has two problems:
   - The starting date should start from `'2016-01-01'` and not `'2016-01-10'`.
   - The ending should include the data `'2019-12-31'`. (change `<` to `<=`)